### PR TITLE
DOC-1925: Add Skins and Icons examples to examples.adoc index page

### PR DESCRIPTION
**Pull request by FarzadHayat**
Ticket: DOC-1925

Changes:
* Added Skins and Icons examples section to the index page

Pre-checks:
- [x] Branch prefixed with `feature/6/` or `hotfix/6/`
- [x] Changelog entry added
- [x] `modules/ROOT/nav.adoc` has been updated (if applicable)

Review:
- [x] Documentation Team Lead has reviewed

### CODE REVIEW

**Review by EkimChau — APPROVED** [review]

**Review by MitchC1999 — APPROVED** [review]

File: modules/ROOT/partials/index-pages/skins-and-icons-examples.adoc [hunk 1]
```diff
@@ -0,0 +1,81 @@
+This table sets out {productname}'s range of pre-made skins and icon packs included in the {productname} documentation.
+
+[cols="1,1"]
+|===
+
+a|
+[.lead]
+xref:bootstrap-demo.adoc[Bootstrap Demo]
+
+Example form with colors, forms, and buttons matching Bootstrap framework UI.
```
Comment (kemister85) [inline modules/ROOT/partials/index-pages/skins-and-icons-examples.adoc:10]:
```suggestion
Example form with colors, forms, and buttons to closely match the default Bootstrap framework UI.
```

File: modules/ROOT/partials/index-pages/skins-and-icons-examples.adoc [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+a|
+[.lead]
+xref:borderless-demo.adoc[Borderless Demo]
+
+Default TinyMCE skin without outer border, ideal for "fullscreen" editing.
```
Comment (kemister85) [inline modules/ROOT/partials/index-pages/skins-and-icons-examples.adoc:16]:
```suggestion
Default TinyMCE skin without an outer border, ideal for "fullscreen" editing.
```

**Review by kemister85 — APPROVED** [review]
@FarzadHayat Check with @MitchC1999 on the status of `staging` before we merge this ticket, so we dont cause any issues with his current docs-migration project.

- [x] @MitchC1999 confirmed `staging` is 🟢

File: changelog.md [hunk 3]
```diff
@@ -6,6 +6,10 @@ The format is loosely based on [Keep a Changelog](https://keepachangelog.com/en/
 
 ### Unreleased
 
+### TBA
```
Comment (FarzadHayat) [inline changelog.md:9]: ### 2024-01-18

**Review by FarzadHayat — COMMENTED** [review]
Applied code review changes and updated changelog.md

**Pull request merged**